### PR TITLE
update release process document

### PR DESCRIPTION
**Pull request by liunaijie**

### CODE REVIEW

**Comment by liunaijie** [comment]
@Hisoka-X @TyrantLucifer I update the current release process document, PTAL.

**Review by Hisoka-X — DISMISSED** [review]
LGTM. Thanks @liunaijie ! cc @arshadmohammad for a look.

File: community/contribution_guide/release.md [hunk 1]
```diff
@@ -385,86 +403,75 @@ Checklist for reference:
...
-[RESULT] [VOTE] Release Apache SeaTunnel 2.3.3
+[RESULT] [VOTE] Release Apache SeaTunnel 2.3.8
 
 Hi SeaTunnel community,
 
 This vote now closes since 72 hours have passed.
 
 The vote PASSES with
 
-3 (+1 binding) votes from the IPMC,
```
Comment (arshadmohammad) [inline community/contribution_guide/release.md:417]: Can we have below format

There are 6 approving votes, 4 of which are binding:
- Jia Fan (binding)
- Guangdong Liu (binding)
- hailin0 (binding)
- David (binding)
- Mohammad Arshad
- User name 6


2 (+0 no opinion) vote
- User name 7
- User name 8

There are no disapproving votes

The vote passes with 4 binding +1 votes and 2 non-binding +1 votes

File: community/contribution_guide/release.md [hunk 2]
```diff
@@ -385,86 +403,75 @@ Checklist for reference:
...
-
-Hi SeaTunnel Community,
-
-Thanks, everyone, I will close this vote thread and the results will be tallied.
-
-Best wishes!
-Chao Tian
-#### 归票
-[RESULT] [VOTE] Release Apache SeaTunnel 2.3.3
+[RESULT] [VOTE] Release Apache SeaTunnel 2.3.8
```
Comment (arshadmohammad) [inline community/contribution_guide/release.md:416]: Lets start with release condidate 0
Change "[VOTE] Release Apache SeaTunnel 2.3.8" to "[VOTE] Release Apache SeaTunnel 2.3.8 condidate 0"

File: community/contribution_guide/release.md [hunk 3]
```diff
@@ -337,31 +351,35 @@ svn --username=${APACHE LDAP USERNAME} commit -m "release ${RELEASE.VERSION}"
 ### dev@seatunnel.apache.org Voting
 #### Voting initiate
 ```bash
-[VOTE] Release Apache SeaTunnel 2.3.3
+[VOTE] Release Apache SeaTunnel 2.3.8
```
Comment (arshadmohammad) [inline community/contribution_guide/release.md:354]: Lets start with release condidate 0
Change "[VOTE] Release Apache SeaTunnel 2.3.8" to "[VOTE] Release Apache SeaTunnel 2.3.8 condidate 0"

File: community/contribution_guide/release.md [hunk 4]
```diff
@@ -385,86 +403,75 @@ Checklist for reference:
...
-Chao Tian
 ```
 
 
 ## Official Release
 ### Moving files
-Moving files from the dev repository to the release repository.
+Moving files from the dev repository to the release repository. This step only PMC member has permission.
 ```bash
 svn mv https://dist.apache.org/repos/dist/dev/incubator/seatunnel/${RELEASE.VERSION} https://dist.apache.org/repos/dist/release/incubator/seatunnel/
```
Comment (Hisoka-X) [inline community/contribution_guide/release.md:455]:
```suggestion
svn mv https://dist.apache.org/repos/dist/dev/seatunnel/${RELEASE.VERSION} https://dist.apache.org/repos/dist/release/seatunnel/
```

File: community/contribution_guide/release.md (hunk 3 above)
Comment (Hisoka-X) [reply community/contribution_guide/release.md:354]: +1

Comment (Hisoka-X) [reply community/contribution_guide/release.md:354]: how about start first vote with `[VOTE] Release Apache SeaTunnel 2.3.8 (RC1)`

Comment (arshadmohammad) [reply community/contribution_guide/release.md:354]: this is also ok. we can have RC1

Comment (liunaijie) [reply community/contribution_guide/release.md]: Done

File: community/contribution_guide/release.md (hunk 4 above)
Comment (liunaijie) [reply community/contribution_guide/release.md]: Updated

**Review by arshadmohammad — APPROVED** [review]
LGTM +1

**Comment by Hisoka-X** [comment]
waiting https://github.com/apache/seatunnel/pull/7882 to fix test case.

**Pull request merged**